pkg/client/unversioned/horizontalpodautoscaler.go

```diff
@@ -94,6 +95,13 @@ func (c *horizontalPodAutoscalers) Update(horizontalPodAutoscaler *experimental.
 	return
```

Comment:
FYI, at line 77 above, you may need to do 
```
body, err := api.Scheme.EncodeToVersion(options, latest.GroupOrDie("").GroupVersion)
```
xref: https://github.com/kubernetes/kubernetes/pull/14566/files#r40605889

No need to do in this PR, but could help if you are debugging issues.